types/src/model/blockchain.rs

```diff
@@ -327,6 +346,42 @@ pub struct GetBlockStats {
     pub utxo_size_increase_actual: Option<i32>,
 }
 
+/// Models the result of JSON-RPC method `getchainstates`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct GetChainStates {
+    /// The number of headers seen so far.
+    pub headers: u32,
+    /// List of the chainstates ordered by work, with the most-work (active) chainstate last.
+    pub chain_states: Vec<ChainState>,
+}
+
+/// A single chainstate returned as part of `getchainstates`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct ChainState {
+    /// Number of blocks in this chainstate.
+    pub blocks: u32,
+    /// Blockhash of the tip.
+    pub best_block_hash: BlockHash,
+    /// nBits: compact representation of the block difficulty target.
+    pub bits: Option<String>, // v29 and later only.
+    /// The difficulty target.
+    pub target: Option<String>, // v29 and later only.
```

Comment:
```suggestion
    pub target: Option<Target>, // v29 and later only.
```

In 91a0e2f

Grep for 'target: ' to see other instances of this.